main.css

```diff
@@ -1 +1,13 @@
 // FEED ME YOUR BEST CSS STYLES THANKS <3
+
+.titanic {
+ float: none; 
+}
+
+.weird {
+ display: flex; 
+}
+
+.okay {
```

Comment:
Henlo Matsu, thank you for your request, but can you delete this one part, we consider this to be a hate crime. thanks no hate only love

Reply:
Hello hekekee, I am very sorry. DId not mean to hate just love <3